pkg/catalog/routes.go

```diff
@@ -460,3 +460,48 @@ func (mc *MeshCatalog) GetServicesForServiceAccounts(saList []service.K8sService
 
 	return serviceList
 }
+
+// GetHostnamesForUpstreamService returns the hostnames over which an upstream service is accessible from a downstream service
+// The hostname is the FQDN for the service, and can include ports as well.
+// Ex. bookstore.default, bookstore.default:80, bookstore.default.svc, bookstore.default.svc:80 etc.
+// TODO: replace GetResolvableHostnamesForUpstreamService with this func once routes refactor is complete (#issue)
```

Comment:
is there any issue number?

Reply:
#2048

Reply:
Okay, I think that is missing from line 467 but it can be added later